resources/definitions/sovol_sv06_plus.def.json

```diff
@@ -0,0 +1,71 @@
+{
+    "version": 2,
+    "name": "Sovol-SV06 Plus",
+    "inherits": "sovol_base_planetary",
+    "metadata":
+    {
+        "visible": true,
+        "author": "ashokvarma.com",
+        "quality_definition": "sovol_base_planetary"
+    },
+    "overrides":
+    {
+        "acceleration_wall":{ "value" : "acceleration_print / 4"},
+        "bottom_layers": { "value": 4 },
+        "coasting_enable": { "value": true },
+        "fill_outline_gaps": { "value": true },
+        "gantry_height": { "value": 35 },
+        "infill_overlap": { "value": 10.0 },
+        "infill_pattern": { "value": "'gyroid'" },
+        "infill_wipe_dist": { "value": 0.1 },
+        "jerk_travel": { "value": "jerk_print" },
+        "machine_depth": { "default_value": 300 },
+        "machine_end_gcode": { "default_value": "G91 ;Relative positioning\nG1 E-2 F2700 ;Retract a bit\nG1 E-2 Z0.2 F2400 ;Retract and raise Z\nG1 X0 Y220 F3000 ;Wipe out\nG1 Z10 ;Raise Z more\nG90 ;Absolute positionning\n\nG1 X0 Y{machine_depth} ;Present print\nM106 S0 ;Turn-off fan\nM104 S0 ;Turn-off hotend\nM140 S0 ;Turn-off bed\n\nM84 X Y E ;Disable all steppers but Z\n" },
+        "machine_head_with_fans_polygon":
+        {
+            "default_value": [
+                [-44, 16],
+                [-44, -60],
+                [44, -60],
+                [44, 16]
+            ]
+        },
+        "machine_height": { "default_value": 340 },
+        "machine_max_feedrate_z": { "value": 20 },
```

Comment:
Max feedrate on the Z-axis seems to be `10`

https://github.com/Sovol3d/SV06-PLUS/blob/3569ae18457dd8a7447e3bcdbe7a965e8be760be/MarlinSV06Plus/Marlin/Configuration.h#LL940C22-L940C22